HW12-Solid/src/main/java/ru/sinara/atm/exception/AtmException.java

```diff
@@ -0,0 +1,7 @@
+package ru.sinara.atm.exception;
+
+public class AtmException extends Exception {
```

Comment:
Лучше использовать непроверяемые исключения.
```suggestion
public class AtmException extends RuntimeException {
```